codegenerator/cli/templates/static/codegen/src/eventFetching/hyperfuel/HyperFuel.res

```diff
@@ -89,7 +89,10 @@ module LogsQuery = {
   ): HyperFuelClient.QueryTypes.query => {
     {
       fromBlock,
-      toBlockExclusive: toBlockInclusive + 1,
+      toBlockExclusive: ?switch toBlockInclusive {
+        | Some(toBlockInclusive) => Some(toBlockInclusive + 1)
+        | None => None
+      },
```

Comment:
I tried to compare the latest envio version with this one. Looked at TUI of the Fuel testnet. I don't really see any difference, but I think it'll be more noticeable when HyperSync is lagging and takes more time to respond.

Reply:
Not related but maybe it would be worth firing that hook on a throttler as well. Makes the TUI less jittery

Reply:
Probably firing the hook as we do and throttle the render itself.

Reply:
Sounds like a good idea